lib/pages/guide.js

```diff
@@ -1,11 +1,14 @@
-import unified from 'unified';
+import directive from 'remark-directive';
+import footnotes from 'remark-footnotes';
+import h from 'hastscript';
 import md2mdast from 'remark-parse';
+import rehype2html from 'rehype-stringify';
+import remark2rehype from 'remark-rehype';
 import slug from 'remark-slug';
 import smartypants from '@silvenon/remark-smartypants';
-import footnotes from 'remark-footnotes';
 import toc from 'mdast-util-toc';
-import remark2rehype from 'remark-rehype';
-import rehype2html from 'rehype-stringify';
+import unified from 'unified';
+import visit from 'unist-util-visit';
```

Comment:
I'm guessing you have some kind of autoformatter running on your IDE, huh

Reply:
No, it bothered me they weren't sorted, so I did it manually